test/common.jl

```diff
@@ -29,6 +29,8 @@ generic_network_data = JSON.parse("""{
 
 generic_network_time_series_data = Dict(
     "num_steps" => 3,
+    "global_constant" => ℯ,
```

Comment:
nit, lets just use a number here rather than a numerical constant that looks very much like a character.

Reply:
Done